libp2p/Cargo.toml

```diff
@@ -91,6 +91,11 @@ webtransport-websys = ["dep:libp2p-webtransport-websys"]
 yamux = ["dep:libp2p-yamux"]
 upnp = ["dep:libp2p-upnp"]
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+libp2p-webrtc-websys = { workspace = true, optional = true }
+libp2p-websocket-websys = { workspace = true, optional = true }
+libp2p-webtransport-websys = { workspace = true, optional = true }
+
```

Comment:
We should move this after the dependencies but before `target.'cfg(not(target_arch = "wasm32"))'.dependencies` section.

Reply:
You got it